### PR TITLE
Add T Flip-Flop

### PR DESCRIPTION
**Pull request by stefanklinge**

### CODE REVIEW

**Comment by stefanklinge** [comment]
A new T_Flip_Flop

File: common/d_flip_flop/t_flip_flop.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <meta name="implementitem" value="tflipflop" />
+    <flags>
+      <option>horizontalonly</option>
+      <option>middlemustalign</option>
+    </flags>
+  </declaration>
+  <connections>
+    <group>
+      <connection name="t" start="_Start-20y" end="_Middle-20x-20y" edge="start" />
+      <connection name="ck" start="_Start" end="_Middle-20x" edge="start" />
```
Comment (SamuelFisher) [inline common/d_flip_flop/t_flip_flop.xml]: This needs to move down a bit to line up with the clock input.

```suggestion
      <connection name="ck" start="_Start+20y" end="_Middle-20x+20y" edge="start" />
```

File: common/d_flip_flop/t_flip_flop.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component version="1.2" xmlns="http://schemas.circuit-diagram.org/circuitDiagramDocument/2012/component/xml">
+  <declaration>
+    <meta name="name" value="T Flip-Flop" />
```
Comment (SamuelFisher) [inline common/d_flip_flop/t_flip_flop.xml]: All components need a unique ID. I've generated one here if you want to use this:

```suggestion
    <meta name="name" value="T Flip-Flop" />
    <meta name="guid" value="9a442c47-4909-4ced-a77e-9d2d9b244e7f" />
```

File: .github/ISSUE_TEMPLATE/component-request.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
```
Comment (SamuelFisher) [inline .github/ISSUE_TEMPLATE/component-request.md]: Please could you delete this file?

**Comment by SamuelFisher** [comment]
https://www.circuit-diagram.org/components/9a442c47-4909-4ced-a77e-9d2d9b244e7f

**Pull request merged**